Gopkg.toml

```diff
@@ -22,6 +22,8 @@
 # TODO: fork the repo to checkin artifacts, or find an alternative repo that has artifacts available.
 # This repo includes only istio-generated files.
 
+[metadata]
```

Comment:
oh ? what's that ?

Reply:
https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#metadata

Reply:
nice, can you check the effect on inputs-digest
https://github.com/istio/istio/blob/master/Gopkg.lock#L1469